src/field_theory/normal.lean

```diff
@@ -6,13 +6,14 @@ Authors: Kenny Lau
 
 import field_theory.minpoly
 import field_theory.splitting_field
-import linear_algebra.finite_dimensional
+import field_theory.tower
+import ring_theory.power_basis
 
 /-!
 # Normal field extensions
 
 In this file we define normal field extensions and prove that for a finite extension, being normal
-is the same as being a splitting field (TODO).
+is the same as being a splitting field.
```

Comment:
Strictly, don't you need to state the `iff` for this to be true? Or do we already have `is_splitting_field.of_normal` elsewhere?

Reply:
We already have it, further up in the file: `normal.exists_is_splitting_field`

Reply:
Maybe worth linking those then:
```suggestion
is the same as being a splitting field (`normal.of_is_splitting_field` and `normal.exists_is_splitting_field`).
```

An `iff` lemma might still be handy if someone wants to use `rw` in a proof.

Reply:
Adding explicit names in the docs is always a good idea. The `iff`-lemma is fine, but because these are typeclasses, I don't think such `rw` should be common (it's probably a sign that you're fighting against the system).

Reply:
So should I add an `iff`-lemma?